gpflow/conditionals.py

```diff
@@ -101,7 +103,7 @@ def _conditional(Xnew, X, kern, f, *, full_cov=False, q_sqrt=None, white=False,
     :param X: data points, size M x D.
     :param kern: GPflow kernel.
     :param f: data matrix, M x R, representing the function values at X,
-        for K functions.
+        for R functions.
```

Comment:
Where is R definition? Do we have it in glossary?

Reply:
Just above : 
`:param f: data matrix, M x R,...`
definitely no K!

Reply:
It is sort of the implicit letter for the usual `num_latent`